src/otx/algorithms/common/configs/training_base.py

```diff
@@ -198,6 +198,15 @@ class BaseLearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING,
         )
 
+        input_size = selectable(
+            default_value=InputSizePreset.DEFAULT,
+            header="Change input image size.",
+            description="You can reduce training and inference time by using small input size.",
+            warning="Small input size may decrease model performance.",
```

Comment:
Just a question: should we specify these duplicated info here again? Can we just inherit somehow? BTW, please align the descriptions.

Reply:
oh maybe it is already inherited, so it isn't necessary. I'll check it.

Reply:
I tried, but it doesn't work. I'll align all sentences with updated one.